### PR TITLE
refactored a few things to speed up file checking

### PR DESCRIPTION
**Pull request by sandman45**
added better cleanup for the m3u8

### CODE REVIEW

File: Radiant/fileHandler.js [hunk 1]
```diff
@@ -45,40 +45,39 @@ module.exports.watcher = (ouPath, args) => {
...
-                        } else {
-                            console.log(`File not found ${err}`);
-                        }
-                    });
-                }
-            });
-        });
-    }, 1500);
-
+  //   setTimeout(() => {
```
Comment (laynemoseley) [inline Radiant/fileHandler.js]: can this be removed completely?

File: Radiant/fileHandler.js [hunk 2]
```diff
@@ -90,14 +89,11 @@ const checkM3U8 = (file) => {
...
                 if(line === '#EXT-X-ENDLIST\n'){
-                    console.log(`#EXT-X-ENDLIST => ${file}`);
-                    console.log(`Deleting file => ${file}`);
-                    fs.unlink(file, (err) => {
-                        if(err){
-                            console.log(`ERROR: File Not Found ${err.message}`);
-                        }
-                        delete streamTracker[file];
-                    });
+                    // console.log(`#EXT-X-ENDLIST => ${file}`);
```
Comment (laynemoseley) [inline Radiant/fileHandler.js]: need these too?

File: Radiant/fileHandler.js [hunk 3]
```diff
@@ -112,43 +108,33 @@ const checkM3U8 = (file) => {
...
+                    uploadFile(info, false);
+                } else {
+                    checkFile(info, fileInfo.size);
                 }
-            });
-        }, 1000, [{
-            info
-        }]);
+            }  else {
+                console.log(`File not found ${err}`);
```
Comment (laynemoseley) [inline Radiant/fileHandler.js]: is this an error state we should actually keep track of?

File: Radiant/fileHandler.js [hunk 4]
```diff
@@ -112,43 +108,33 @@ const checkM3U8 = (file) => {
...
     }
 };
 
 /**
  * uploadFile
  * @param info
+ * @param endStream
  */
-const uploadFile = function (info){
+const uploadFile = function (info, endStream){
```
Comment (laynemoseley) [inline Radiant/fileHandler.js]: `endStream` means that this it should finish the m3u8 file right?

File: Radiant/fileHandler.js (hunk 3 above)
Comment (sandman45) [reply Radiant/fileHandler.js]: no

File: Radiant/fileHandler.js (hunk 4 above)
Comment (sandman45) [reply Radiant/fileHandler.js]: correct when this is set to true it will upload the m3u8 and then delete it..

File: Radiant/fileHandler.js (hunk 1 above)
Comment (sandman45) [reply Radiant/fileHandler.js]: yes i removed that and the function it was in

**Pull request merged**